tools/peptideshaker/peptide_shaker.xml

```diff
@@ -135,7 +135,7 @@
             && unzip \$cwd/peptideshaker_output.zip 'reports/*' -d \$cwd/output_reports
         #end if
 
-	      #if len(output_reports_list)>0:
+        #if len(output_reports_list)>0:
```

Comment:
mh, this indentation is off and do we miss a $ here?

Reply:
As far as I can seen in the github editor indentation is now correct. 

good point with the `$` .. wonder why this works also without.